backend/src/services/sync/museum/treasureWriter.test.ts

```
@@ -467,3 +480,142 @@ describe('what a run stores about a work photograph', () => {
+
+/**
+ * A gated source may not overwrite what a reader can already see, and since
+ * ADR-0037 that covers a work's fields as it has always covered the museum's
+ * own. Measured before this existed: run 64 rewrote the attribution of The Wine
+ * Glass (Gemäldegalerie) from Johannes Vermeer to an obscure namesake, live,
+ * under a gated category, with nobody asked (#717). The row's own state decides
+ * visibility — a work verified through one venue is on show there even where
+ * another venue's link is still pending.
+ */
+describe('a visible work under a gated source', () => {
+  const FILE = 'http://commons.wikimedia.org/wiki/Special:FilePath/Wine%20Glass.jpg';
```

ESLint

[error] Using http protocol is insecure. Use https instead.

(sonarjs/no-clear-text-protocols)